collections/_article/introducing-the-improved-ufbx-importer-in-godot-4-3.md

```diff
@@ -0,0 +1,55 @@
+* [#88824 (Retargeting option to use a template for silhouette)](https://github.com/godotengine/godot/pull/88824): This feature can be used to reference the RESET animation of a known good (T-pose) reference file. Having a separate reference pose .fbx is a common practice in some animation sets.
+* [#88821 (Allow preserving the initial bone pose in rest fixer)](https://github.com/godotengine/godot/pull/88821): This allows for importing a retargeted skeleton while keeping it in its initial pose.
+* [#88819 (Add new scene import option to import as Skeleton)](https://github.com/godotengine/godot/pull/88819): This solves cases especially common in .fbx files without any meshes, where animated skeletons import as a series of nodes.
+* Improvements to bone detection [#90050 (Add tips detection to auto mapping in bone mapper)](https://github.com/godotengine/godot/pull/90050): Improves support for Mixamo and ReadyPlayerMe finger bone conventions
+* Several other bugfixes ([#90019](https://github.com/godotengine/godot/pull/90019), [#90064](https://github.com/godotengine/godot/pull/90064), [#90065](https://github.com/godotengine/godot/pull/90065), [#91641](https://github.com/godotengine/godot/pull/91641), [#92012](https://github.com/godotengine/godot/pull/92012)) to retargeting and animation import.
+
+# Compatibility with FBX2glTF and other versions of Godot Engine
+
+After upgrading to Godot 4.3, existing files in your project will continue to import with FBX2glTF. By default, newly added files will import with ufbx. This default may be changed by using the "Preset" button near the top of the Import dock, or in the "Import Defaults" pane of Project Settings.
+
+Use extreme caution when changing the importer of an existing .fbx from FBX2glTF to ufbx: the two will have different node hierarchies, which will break modifications made in instanced scenes, NodePath references in scripts, affect rest poses of skeletons and possibly mix up surface override materials due to the change in surface ordering.
+
+In accordance with [our release policy](https://docs.godotengine.org/en/stable/about/release_policy.html#what-are-the-criteria-for-compatibility-across-engine-versions), we will continue to support projects which have chosen to use FBX2glTF throughout the 4.x cycle for compatibility with existing imports. ufbx will affect new 3d imports by default, but the importer for .fbx files may be changed in the Import dock.
+
+# Issues and testing
+
+Samuli has spent exceptional time and effort testing the ufbx library on tens of thousands of .fbx test cases from a variety of sources, and we are proud to consider the new ufbx importer feature-complete. (This level of rigorous design and testing is a stark contrast to previous .fbx importer rewrites some of you may recall from Godot 3. We can now be confident when we say that the importer is a feature-complete implementation of .fbx support for Godot.)
```

Comment:
```suggestion
Samuli has spent exceptional time and effort testing the ufbx library on tens of thousands of .fbx test cases from a variety of sources, and we are proud to consider the new ufbx importer feature-complete. This level of rigorous design and testing is a stark contrast to previous .fbx importer rewrites some of you may recall from Godot 3.
```
Removed last sentence as it seemed redundant (you've already just said it's feature-complete). Also, I didn't think the brackets were necessary.